docker/metrics-collector/prometheus/prometheus.yml

```diff
@@ -0,0 +1,8 @@
+global:
+  scrape_interval:     5s
+  evaluation_interval: 5s
+
+scrape_configs:
+  - job_name: 'wnode'
+    static_configs:
+      - targets: ['docker.for.mac.localhost:32794', 'docker.for.mac.localhost:32792']
```

Comment:
That is a Mac-specific alias. Do we want to support Linux?

Reply:
+1, also why 3rd node is missing?

Reply:
This is the most tricky parts. First of all, as `--scale` is used, there is no way to control the number of nodes in a static config. Secondly, Linux should be totally supported. The best would be some service discovery to do that but I haven't researched how to do that in Prometheus.

I will probably add a comment that this should be changed if someone wants to use it unless you already know how to do it properly. Also, we can improve it later step by step.